src/7-to-8/major-changes/cylc-install.rst

```diff
@@ -89,7 +89,7 @@ Remote Installation
 
 Remote file installation does not occur until running the workflow.
 When the first task runs on a remote platform, Cylc will transfer files from
-the :term:`run directory` to the install target.
+the :term:`run directory` to the :term:`install target <install target>`.
```

Comment:
Why the difference in syntax for `run directory` vs `install target`?

Reply:
> Why the difference in syntax for `run directory` vs `install target`?
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax

The `<>` is for linking to a title, different to what is displayed. In this case it is redundant as install target is the same, typed out of habit.